includes/Ajax.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace Wedevs\DokanMigrator;
+
+use Exception;
+use Wedevs\DokanMigrator\Helpers\MigrationHelper;
+
+/**
+ * Ajax request handler class.
+ *
+ * @since 1.0.0
+ */
+class Ajax {
+
+    /**
+     * Class constructor.
+     *
+     * @since 1.0.0
+     */
+    public function __construct() {
+        add_action( 'wp_ajax_dokan_migrator_count_data', array( $this, 'count' ) );
+        add_action( 'wp_ajax_dokan_migrator_import_data', array( $this, 'import' ) );
+        add_action( 'wp_ajax_dokan_migrator_last_migrated', array( MigrationHelper::class, 'get_last_migrated' ) );
+        add_action( 'wp_ajax_dokan_migrator_active_vendor_dashboard', array( MigrationHelper::class, 'active_vendor_dashboard' ) );
+    }
+
+    /**
+     * Count the data of vendor or order.
+     *
+     * @since 1.0.0
+     *
+     * @return void
+     */
+    public function count() {
+        $this->verify_nonce( $_REQUEST ); // phpcs:ignore
+
+        $request = wp_unslash( $_REQUEST ); // phpcs:ignore
```

Comment:
Anywhere it has been done this way, must be removed. No need to `unslash` the whole `$_REQUEST` and store it to another variable. Just do the process as follows:
```
$import = ! empty( $_POST['import'] ) ? sanitize_text_field( wp_unslash( $_POST['import'] ) ) : 'vendor';
$migratable = ! empty( $_POST['migratable'] ) ? boolval( wp_unslash( $_POST['migratable'] ) ) : false;
```

Reply:
Also, why are the posted values parsed using `$_REQUEST`?
Try to avoid this if not necessary. If not the process is taking any `$_GET` variable, then `$_POST` is appropriate for `Ajax` calls.